src/MalformedURIException.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GuzzleHttp\Psr7;
```

Comment:
maybe this should go in `GuzzleHttp\Psr7\Exception`

Reply:
Good point, done!